composer.json

```
@@ -53,7 +53,7 @@
         "doctrine/dbal":                        "^2.10",
         "doctrine/doctrine-module":             "^4.0",
         "doctrine/orm":                         "^2.6.4",
-        "doctrine/persistence":                 "^1.3",
+        "doctrine/persistence":                 "^1.3.3|^2.0",
```

See https://github.com/doctrine/DoctrineORMModule/pull/644#discussion_r537365861
```suggestion
        "doctrine/persistence":                 "^1.3 || ^2.0",
```